From e1228ee0158e4876b2b051705dd485e51060595b Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@nic.cz>
Date: Tue, 21 Apr 2026 19:19:55 +0200
Subject: [PATCH] tio: Fix compilation error based on missing nls -lintl

Fix compilation error due to missing nls library.
Fixing error:
../arm-openwrt-linux-muslgnueabi/bin/ld.bfd: cannot find -lintl: No such file or directory
---
 utils/tio/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/utils/tio/Makefile b/utils/tio/Makefile
index 2382e03e0..811b0671d 100644
--- a/utils/tio/Makefile
+++ b/utils/tio/Makefile
@@ -23,6 +23,7 @@ PKG_BUILD_FLAGS:=no-mips16
 
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/meson.mk
+include $(INCLUDE_DIR)/nls.mk
 
 define Package/tio
   SECTION:=utils
@@ -30,7 +31,7 @@ define Package/tio
   SUBMENU:=Terminal
   TITLE:=A simple TTY terminal I/O application
   URL:=https://tio.github.io/
-  DEPENDS:=+lua +glib2
+  DEPENDS:=+lua +glib2 $(INTL_DEPENDS)
 endef
 
 define Package/tio/description
-- 
2.47.3

